src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1230,7 +1230,13 @@ public void describe(ColibriConferenceIQ.ChannelBundle channelBundle)
     @Override
     public void requestKeyframe(long mediaSsrc)
```

Comment:
i kept this one around because i think it still makes sense for Octo, but for a normal endpoint it maps to the same call.  (actually, in `Endpoint` this should probably call the other `requestKeyframe` method, will tweak that).